### DISCUSSION TITLE
Support for isc-kea-dhcp

### DISCUSSION THREAD

**Discussion by Dragao75** [webmin/webmin#1930] [Ideas]
As isc-dhcp is EOL could support for its successor be implemented? (isc-kea) 

many thanks

**Comment by jcameron** [comment]
Does it use the same config file format?

**Comment by Dragao75** [comment]
No,
They switch to json syntax

Regards,

Ronald

Op vr 23 jun 2023 om 00:50 schreef Jamie Cameron ***@***.***>:

**Comment by jcameron** [comment]
Oh wow ... in that case we'd need to re-write the entire DHCP server module unfortunately.

**Comment by wrogner** [comment]
Now nearly a year has passed but there seem to be no advances in this repo.
Q: Any plans as to when (or even) to support ISC Kea in Webmin?

**Comment by NuAngel** [comment]
Any chance we could put a bounty on this and see if people are willing to contribute?

**Comment by bartgrefte** [comment]
A while back I used the contactform on isc.org/contact/ requesting they create a plugin, never got a reply back. Maybe it helps if more people send in requests?

**Reply by NuAngel** [reply]
I just submitted one, too! Worth a kind nudge!

**Comment by cvsmith122** [comment]
Been almost a year since the last update on this request, does any one know if its being worked on ?

**Reply by iliaross** [reply]
> Been almost a year since the last update on this request, does any one know if its being worked on ?

Alright, I’ve been working on it for the past couple of days, and it’s a lot of work to make it a well-rounded Webmin module.

I’ll update this ticket with progress, hopefully before the weekend.

**Reply by cvsmith122** [reply]
Thanks !

**Reply by wrogner** [reply]
I am in no rush any more. Kea is way to complicated for DHCP management. Speed-wise the old solution beats Kea in most scenarios I use.
But still interested in the process.

**Comment by iliaross** [comment]
> In that case we'd need to re-write the entire DHCP server module unfortunately.

Jamie, implemented here https://github.com/webmin/webmin/pull/2736.
